spec/models/concerns/hyrax/solr_document_behavior_spec.rb

```diff
@@ -64,6 +64,17 @@
       it 'resolves the correct model name' do
         expect(solr_document.hydra_model).to eq Monograph
       end
+
+      context 'using non-wings adapter', valkyrie_adapter: :test_adapter do
```

Comment:
I'm not sure if this test is really necessary but it does prove that the method still returns when Wings is disabled.